tempesta_fw/http_msg.c

```diff
@@ -168,6 +168,13 @@ tfw_http_msg_req_spec_hid(const TfwStr *hdr)
 /**
  * Fills @val with second part of special HTTP header containing the header
  * value.
+ *
+ * TODO: this function should be brought to a uniform look with other similar
+ * procedures( @tfw_http_hdr_split() and @tfw_h2_msg_hdr_length()), more
+ * likely - aggregated into one general-purpose procedure, after HTTP/1.1-parser
+ * extending (see also TODO-comment for @tfw_http_hdr_split_cp() procedure); for
+ * now this procedure leaves RWS in the resulting @val and this is not correct
+ * in context of HTTP/2<=>HTTP/1.1 comparisons, transformations etc.
```

Comment:
Just leave a comment to remember that the TODO must be fixed in upcoming PR.

Reply:
TODO is fixed.